build.xml

```diff
@@ -8,50 +8,61 @@
 
     <!-- Target: install-all -->
     <target name="install-all" description="Install all three Drupal sites in the Federated Search Demo">
+        <phingcall target="solr-clear" />
```

Comment:
Just a note -- this adds an interactive prompt to `phing install-all`; is that ok?

Reply:
I think so, I changed the language on that prompt.

Reply:
`drush --existing-config --account-name="admin" --account-pass="admin" --site-name="Federated Search Drupal 8" --root="web/d8/docroot" --yes`

---

` The selected profile has a hook_install() implementation and therefore can not be installed fro  
  m configuration.`